src/pages/Logs/Logs.tsx

```diff
@@ -0,0 +1,9 @@
+import BaseLayout from "layout/BaseLayout/BaseLayout";
+
+const Logs = (): JSX.Element => (
+  <BaseLayout>
+    <h1>Audit logs</h1>
```

Comment:
This can be `<Header>Audit logs</Header>`

Reply:
Done.